addons/escoria-core/game/core-scripts/esc/types/esc_statement.gd

```diff
@@ -12,9 +12,12 @@ signal interrupted(return_code)
 # The list of ESC commands
 var statements: Array = []
 
-# Indicated whether this event was interrupted.
+# Indicates whether this event was interrupted.
 var _is_interrupted: bool = false
 
+# Indicates whether this event was finished.
+var _is_finished: bool = false
```

Comment:
Hm. I *think*, this should be noted as public and not private, because it is modified in an object instance in line 48, although the object instance is the same as the object.

I mean, GDScript doesn't support private/public variables anyway, but still...

Reply:
Alright! Fixed